src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -198,7 +195,23 @@ public void setHasLongFeedbackText(boolean hasLongFeedbackText) {
     }
 
     @JsonIgnore
-    public LongFeedbackText getLongFeedbackText() {
+    public Optional<LongFeedbackText> getLongFeedback() {
+        return getLongFeedbackText().stream().findAny();
```

Comment:
This seems strange / wrong. If there are actually more than 1 `LongFeedbackText` Objects, just returning any object could lead to flaky behavior.